pkg/controller/garbagecollector/metrics.go

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
The code is using a dynamic client, which doesn't expose the embedded RESTClient. The client metric is not that important currently, I'll revisit in the future.